jetty-home/src/main/resources/bin/jetty.sh

```diff
@@ -127,6 +127,7 @@ running()
     return
   fi
   rm -f "$1"
+  rm -f "$2"  
```

Comment:
Odd, not sure what you are trying to do here.

Reply:
Okay. The concept -- and maybe this is not the best approach -- is that if you're going to delete the JETTY_PID then maybe you should also delete the JETTY_STATE. If the PID is deleted and the state file is present then it will cause the JETTY_STATE file to have previous lines still present (and since the grep is looking at the whole file and not the most recent line) it will think it failed.